tests/test_response.py

```diff
@@ -1241,3 +1243,48 @@ def test_cache_expires_set_zero_then_nonzero():
     assert not res.cache_control.no_store
     assert not res.cache_control.must_revalidate
     assert res.cache_control.max_age == 1
+
+def test_default_content_type():
+    class NoDefault(Response):
+        default_content_type = None
+
+    res = NoDefault()
+    assert res.content_type is None
+
+def test_default_charset():
+    class DefaultCharset(Response):
+        default_charset = 'UTF-16'
+
+    res = DefaultCharset()
+    assert res.content_type == 'text/html'
+    assert res.charset == 'UTF-16'
+    assert res.headers['Content-Type'] == 'text/html; charset=UTF-16'
```

Comment:
Yes. This change was made because of this:

```
res = Response()
res.content_type = 'image/jpg'
```

used to keep the content-type parameters, which also meant stuff like:

```
assert res.headers['Content-Type'] == 'image/jpg; charset=UTF-8'
```

This was changed in WebOb 1.6.0. Currently other content-type parameters are copied from the old to the new:

```
res = Response(content_type='application/foo; level=1; charset=UTF-8')
res.content_type = 'image/jpeg'

assert res.headers['Content-Type'] == 'image/jpeg; level=1'
```

Which is going to go away in the future. Content-Type parameters shouldn't be kept when moving from one Content-Type to another.